Header header
std_msgs/String[] labels
std_msgs/String[] properties
std_msgs/String[] affordances
